internal/receiver/lightprometheusreceiver/scraper.go

```diff
@@ -160,6 +160,7 @@ func (s *scraper) convertMetricFamilies(metricFamilies []*dto.MetricFamily, rm p
 			for _, fm := range family.GetMetric() {
 				dp := sum.DataPoints().AppendEmpty()
 				dp.SetTimestamp(now)
+				dp.SetStartTimestamp(now)
```

Comment:
I think the start timestamp is used for [aggregation temporality](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto#L161) so should be the time of the first scrape instead of moving forward?

Reply:
Right, start time for cumulative metrics is the time when scraping started or the cumulative got reset

Reply:
Thanks for the review, fixing